Xiaomi/PocoM4Pro5G/res/xml/power_profile.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">0.1</item>
```

Comment:
With a IPS screen? That's doubtful

Reply:
I couldn't find proper power profile so I just kanged from Poco m3 PR though the one I found were dummies not sure where to look.